prow/cmd/deck/static/user-script.js

```diff
@@ -461,7 +466,8 @@ function createPRCardBody(pr, builds, queries) {
     cardBody.classList.add("mdl-card__supporting-text");
     cardBody.appendChild(title);
     cardBody.appendChild(createJobStatus(builds));
-    cardBody.appendChild(createMergeStatus(pr.Labels.Nodes, queries));
+    const nodes = pr.Labels && pr.Labels.Nodes ? pr.Labels.Nodes : [];
```

Comment:
Not the biggest fan of ternary operators and one liners like this, they're a bit hard to read. Style nit though, so meh.

/lgtm
/hold cancel
/woof